### PR TITLE
Fix typos and grammatical errors in contribution guidelines

### PR DESCRIPTION
**Pull request by IldySilva**

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -39,7 +39,7 @@ In case you couldn't find any issue, it's better to create an issue to explain w
 We have a *_chart_painter.dart class per each chart type. It draws elements into the Canvas.
 We made the CanvasWrapper class, because we wanted to test draw functions.
 CanvasWrapper class holds a canvas and all draw functions proxies through it.
-You should use it for drawing on the canvas, Instead of direct accessing to canvas.
+You should use it for drawing on the canvas,Instead of directly accessing the canvas.
```
Comment (imaNNeo) [inline CONTRIBUTING.md:42]: I think you removed the space after the comma by a mistake

**Comment by imaNNeo** [comment]
Thank you so much for contributing! 
I really appreciate it.

File: CONTRIBUTING.md (hunk 1 above)
Comment (IldySilva) [reply CONTRIBUTING.md]: right,it's now fixed

Comment (imaNNeo) [reply CONTRIBUTING.md]: Thanks!

**Pull request merged**